io_scene_nif/modules/nif_import/armature/__init__.py

```diff
@@ -172,7 +172,7 @@ def mark_armatures_bones(self, ni_block):
 
         # attaching to selected armature -> first identify armature and bones
         elif NifOp.props.skeleton == "GEOMETRY_ONLY" and not self.dict_armatures:
-            b_armature_obj = NifCommon.SELECTED_OBJECTS[0]
```

Comment:
If we are moving to use bpy.context, then can this be removed from NifCommon.
Also would this be affected by operators that don't use a context copy or that are setting objects selected?

Reply:
It was just a hotfix, since getting it from nifcommon did not work ol